src/activities/creatureCreator/sketches/index.js

```diff
@@ -1,18 +1,33 @@
+import { CANVAS_WIDTH, CANVAS_HEIGHT, CANVAS_BG_COLOR } from "../utils";
+
 const sketch = p5 => {
+  let eyes;
+
   p5.setup = () => {
-    p5.background(220);
+    p5.createCanvas(CANVAS_WIDTH, CANVAS_HEIGHT);
+    p5.background(CANVAS_BG_COLOR);
   };
 
-  p5.draw = () => {
-    // console.log("draw!");
-  };
+  const draw = () => {
```

Comment:
i refactored this so that we only re-draw the canvas when we need to (i.e., methods that require a canvas draw will need to call this method, rather than it being called every frame). we may not want this in the future, but it saves us a ton of draw cycles for now